ui/packages/consul-ui/app/styles/base/color/README.mdx

```diff
@@ -0,0 +1,81 @@
+# Colors
+
+All colors for the immediate future should _mostly_ use the following `tone`
+CSS properties, unless:
+
+1. We get around to eventually adding semantic colors names (or adding something else) in which case these docs should be updated.
+2. You are absolutely sure the color you want should never be changed by a theme. These should be very very few and far between so even if we get that wrong it should be easy to update.
+
+`tones` should always using the following form:
+
+```css
+property: rgb(var(--tone-gray-500));
+```
+
+In other words always use `rgb` this gives us the flexibility to easily use thee variables with alpha if we ever need to.
```

Comment:
typo: `thee` should be `these` (i think)

Reply:
```suggestion
In other words always use `rgb` this gives us the flexibility to easily use these variables with alpha if we ever need to.
```